ui/pages/settings/advanced-tab/advanced-tab.component.js

```diff
@@ -490,6 +492,38 @@ export default class AdvancedTab extends PureComponent {
     );
   }
 
+  renderDismissSeedBackupReminderControl() {
+    const { t } = this.context;
+    const {
+      dismissSeedBackUpReminder,
+      setDismissSeedBackUpReminder,
+    } = this.props;
+
+    return (
+      <div
```

Comment:
just as an aside to any other engineers who review this and go "dang why did Brad not recommend Typography/Box components here" -- these styles and HTML are all pre-existing or copied from other toggles. At some point in the future we should simplify the settings pages but at present this is a-ok 🆗